src/Runner.Worker/Worker.cs

```diff
@@ -59,6 +59,9 @@ public async Task<int> RunAsync(string pipeIn, string pipeOut)
                         channelMessage = await channel.ReceiveAsync(csChannelMessage.Token);
                     }
 
+                    // This is useful for validating that the message wasn't truncated when being sent over the anonymous pipe.        
+                    Trace.Info($"Job Message char length: '{channelMessage.Body.Length}', byte length '{System.Text.Encoding.UTF8.GetByteCount(channelMessage.Body)}'.");
```

Comment:
Even just this would be very useful. Can we merge this?

Reply:
nit: do we need this now that ProcessChannel.ReceiveAsync will traces?

Reply:
Nope!